osu.Game/Screens/Play/Player.cs

```diff
@@ -145,10 +149,17 @@ protected override void OnEntering(GameMode last)
             base.OnEntering(last);
 
             (Background as BackgroundModeBeatmap)?.BlurTo(Vector2.Zero, 1000);
+            Background?.FadeTo((100f- dimLevel)/100,1000);
 
             Content.Alpha = 0;
         }
 
+        protected override bool OnExiting(GameMode next)
+        {
+            Background?.FadeTo(1f, 0);

```

Comment:
I actually prefer it instantaneous, but maybe that's because i'm too used to stable.